cloud_info/providers/openstack.py

```diff
@@ -203,6 +203,8 @@ def get_images(self):
                 aux_img['image_os_version'] = distro_version
             if image.metadata.get('image_version', None) is not None:
                 image_version = image.metadata['image_version']
+            elif image.metadata.get('vmcatcher_event_hv_version', None) is not None:
```

Comment:
As reported by the test (you can use tox to run them locally) this line is too long, please fix this: https://travis-ci.org/EGI-FCTF/cloud-info-provider/jobs/247539636
Apart this this seems to be legitimate, thanks for the PR!